src/home.js

```diff
@@ -0,0 +1,36 @@
+const mainDiv = document.getElementById('content');
+const homeSlate = document.createElement('div');
+homeSlate.classList = 'homeSlate blank';
+
```

Comment:
### Optional Change 

- [ ] Implement the [DRY](https://thevaluable.dev/dry-principle-cost-benefit-example/) principle . These lines of code are repeating themselves in almost all modules. Would you mind wring a small script (helpers) and use es6 import method to reuse most of the code that needs reusing in the codebase? 🤔